ui/raidboss/data/03-hw/raid/a3s.js

```diff
@@ -144,12 +152,14 @@
         if (marker1 == marker2) {
           return {
             en: 'Repel: close to ' + data.ShortName(partner),
+            fr: 'Repoussement: Rapprochez-vous de ' + data.ShortName(partner),
```

Comment:
Répulsion. _Repoussement_ does not exist.

Reply:
je corrige